xbmc/utils/StringUtils.cpp

```diff
@@ -591,6 +591,43 @@ std::string& StringUtils::RemoveDuplicatedSpacesAndTabs(std::string& str)
   return str;
 }
 
+bool StringUtils::IsSpecialCharacter(char c)
+{
+  const char specialChars[] = " .-_+,!'\"\t/\\*?#$%&@()[]{}";
+
+  for (char specialChar : specialChars)
+  {
+    if (c == specialChar) {
+      return true;
+    }
+  }
```

Comment:
`std::any_of`use case? Just an idea.

Reply:
It can be done with std::any_of with string_view